etl/glean_etl.py

```diff
@@ -127,6 +127,7 @@ def write_glean_metadata(output_dir, functions_dir, app_names=None):
     annotations_index = requests.get(ANNOTATIONS_URL).json()
     looker_namespaces = yaml.safe_load(requests.get(NAMESPACES_URL).text)
     product_details = requests.get(FIREFOX_PRODUCT_DETAIL_URL).json()
+    latest_fx_release_version = list(product_details)[-1]
```

Comment:
Oh.
While mobile apps follow _roughly_ the same schedule, they don't _exactly_ match.
Let me find out if this is good enough for now or if we have something else to use.

Reply:
at least for android we would have https://product-details.mozilla.org/1.0/mobile_android.json
releases are usually off by a day for reasons™️.
It's a bit more work to parse that, so I'd be ok to just go with the desktop version for now.

We don't have data for iOS anyway, but releases should be similar, so it's ok as well. 
Let's file a followup to gather better data for this.